.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I see `.DS_Store` files in a number of places. What are they? Should they be checked in? My guess is not.

Reply:
deleted the DS_Store stuff.